src/Microsoft.Azure.CosmosRepository/Repositories/DefaultRepository.Paging.cs

```diff
@@ -13,19 +17,36 @@ internal sealed partial class DefaultRepository<TItem>
         int pageSize = 25,
         string? continuationToken = null,
         bool returnTotal = false,
-        CancellationToken cancellationToken = default)
-    {
+        CancellationToken cancellationToken = default)
+        => await PageAsync(
```

Comment:
Minor style nit. Prefer the goes to operator to be on the previous line. Also, if refactoring this entry point, to be an expression-bodied member it's likely we could remove `async` and `await` here to remove an extra state machine blip.

```suggestion
        CancellationToken cancellationToken = default) =>
            await PageAsync(
```

Reply:
- Refactored placement of "goes to"-operator
- Modified passthrough methods to not include async/await